homeassistant/components/climate/__init__.py

```diff
@@ -301,6 +301,10 @@ class ClimateEntity(Entity, cached_properties=CACHED_PROPERTIES_WITH_ATTR_):
     _attr_temperature_unit: str
 
     __mod_supported_features: ClimateEntityFeature = ClimateEntityFeature(0)
+    # Integrations can set `_enable_turn_on_off_backwards_compatibility` to False
+    # to skip the automatic setting of feature flags TURN_ON/TURN_OFF for
+    # backwards compatibility.
```

Comment:
We should rephrase this so it's clear that all integrations should set this flag to False once they've been reviewed and either set (or not set) the TURN_ON / TURN_OFF flags. That will make it clear which integrations have been migrated and which haven't.